package.json

```diff
@@ -30,9 +32,12 @@
   "devDependencies": {
     "babel-jest": "^19.0.0",
     "babel-preset-es2015": "^6.18.0",
+    "commitizen": "2.9.6",
+    "cz-conventional-changelog": "2.0.0",
     "eslint": "^3.11.1",
     "jest": "^19.0.2",
     "pug": "^2.0.0-beta6",
+    "semantic-release": "^6.3.2",
```

Comment:
pls add 
`.yarnrc` with 
```
save-prefix false
```

to make strict dependencies and update all deps to fixed version